secant/Features/BackupFlow/RecoveryPhraseDisplayStore.swift

```diff
@@ -83,6 +83,14 @@ struct RecoveryPhrase: Equatable {
     func toString() -> String {
         words.joined(separator: " ")
     }
+
+    func words(fromMissingIndices indices: [Int]) -> [PhraseChip.Kind] {
+        assert((indices.count - 1) * chunkSize <= self.words.count)
+
+        return indices.enumerated().map({ index, position in
+            .unassigned(word: self.words[(index * chunkSize) + position])
+        })
```

Comment:
Does trailing closure syntax work here?

```suggestion
        return indices.enumerated().map { index, position in
            .unassigned(word: self.words[(index * chunkSize) + position])
        }
```